src/core/renderers/DroppableLayout.tsx

```diff
@@ -46,28 +75,59 @@ const DroppableLayout: React.FC<DroppableLayoutProps> = ({
   renderers,
   cells,
 }) => {
+  const classes = useStyles();
   return (
-    <Grid container direction={direction} spacing={direction === 'row' ? 2 : 0}>
-      {renderLayoutElementsWithDrops(layout, schema, path, renderers, cells)}
+    <Grid container wrap='nowrap' className={classes.droppableLayout} xs>
+      <Grid item key={`${path}-${0}-icon`} className={classes.uiElementIcon}>
+        {getLayoutIcon(layout)}
+      </Grid>
+      <Grid
+        container
+        item
+        direction={direction}
+        spacing={direction === 'row' ? 2 : 0}
+        xs
+      >
+        {renderLayoutElementsWithDrops(
+          layout,
+          schema,
+          path,
+          classes,
+          renderers,
+          cells
+        )}
+      </Grid>
     </Grid>
   );
 };
+const getLayoutIcon = (layout: Layout) =>
+  layout.type === 'HorizontalLayout' ? (
+    <Height style={{ transform: 'rotate(90deg)' }} />
+  ) : (
+    <Height />
+  );
 
 const renderLayoutElementsWithDrops = (
   layout: Layout,
   schema: JsonSchema,
   path: string,
+  classes: ClassNameMap<any>,
```

Comment:
to keep classes typesafety I recommend to use `Record`
```suggestion
  classes: Record<'dropPoint' | 'renderedContent', string>,
```